src/help/pages/help-section/help-section.scss

```diff
@@ -1,2 +1,126 @@
 page-help-section {
+  ion-label {
+    margin: 0 !important;
+  }
+
+  .page-title {
+    padding-top: 12px;
+  }
+
+  .header-container {
+    padding-bottom: 38px;
+  }
+
+  #help-guide-content {
+    padding: 16px;
+  }
+
+  .section-item::after {
```

Comment:
Just looked at the designs, can you do an `&:last-child` to omit the line after the final section?

Reply:
Just had a go myself, doesn't look quite as easy as I made out...

Reply:
I will put an ending on the line instead of removing it.